extensions/vscode/vscode.wake

```diff
@@ -70,6 +70,7 @@ export def vscode _: Result Path Error =
 
     require Pass nodeModules =
         makeExecPlan (npm, "ci", Nil) (packageJSON, packageLockJSON, Nil)
+        | setPlanLabel "Install npm dependencies (npm ci)"
```

Comment:
This additional context is certainly better than we had previously, but (bikeshedding) I wonder if it would be better to put the command/tool first on the line for easy at-a-glance scanning?
```suggestion
        | setPlanLabel "npm ci: Install npm dependencies"
```